Automattic/sensei
# Remove "All" filtering from settings

This PR removes the "All" filter link from the settings that shows all settings simultaneously. It's quite overwhelming to go to the settings page only to be presented with a wall of settings. Plugins like WooCommerce don't do this (although they do use tabs instead of links).

Let's not scare our users, and instead eliminate the ability to view all settings at once. Instead, they'll see the _General_ settings by default when first navigating to the _Settings_.

## Testing
1. Navigate to _Sensei > Settings_.
2. Ensure there is no longer an _All_ filter link.
3. Check that the _General_ filter link is styled as the currently selected filter, and that only the general settings are visible:
![settings](https://user-images.githubusercontent.com/1190420/31127186-1ee5a6c0-a81c-11e7-8bd1-8703692251c1.jpg)
4. Click through the other filters and ensure that only the applicable settings are shown.
5. Go back to _General_ and again ensure that only the general settings are visible.
